### ISSUE TITLE
Random egg generator

### ISSUE THREAD

**Issue by mrmanyk** [CDNRae/pokemon-data-scraper#7]
Hello, 
I originally found these tools from a post which offered a means of generating random pokemon eggs, presumably for egglockes. The website listed is down and while the pokemon-data-scraper suggests it can generate info for the importer, I am seeing no means of doing that. The readme unfortunately doesn't indicate how to use these tool either. 
I would love to have a tool that could generate and import a fat stack of random eggs, the contents of which I would be totally unaware.  If this is the correct tool for that, could you provide a tutorial on how to do this? If not, can you point me in the direction of any tools that can? If not, could I commission you to program this? Especially if one could randomize abilities/natures/tm compatibility eg: Universal randomizer settings. 
Sorry if this isn't the correct place to ask these questions, i just didn't know where else to ask. 

Thanks!

**Comment by CDNRae** [comment]
Hi!

I included a TL;DR at the bottom in case you don't want to read this wall of text (I don't blame you if you don't!). 

Originally yes, that was what this tool was for; it was a website that would generate eggs that could be imported into PKHeX through the use of another tool. However, both the website and the importer tool were a little broken, and they only got worse as PKHeX continued to update. Because of that, I decided to completely overhaul the two tools and make them a little more user-friendly and robust.

This repository, which once held the website, is now a data scraper. It doesn't really do much for anyone who wants to generate eggs -- all it does is pull data from PokeAPI, organize it in a few different ways, and store it in a few different files. I use those files in the Bulk Egg Generator ([here](https://github.com/CDNRae/pkhex-bulk-egg-generator)) to act as a database to generate eggs from. So the average user doesn't need to worry about this data scraper tool; they should be able to do all their egg generating from the Bulk Egg Generator. 

The generator is still being rebuilt though, so I, unfortunately, don't have an update for you right now. However, I'll restore the original website <a href="https://pokemon-egg-generator.herokuapp.com/index">here</a> which you can use with the last release of the Bulk Importer

**TL;DR**: An upcoming update to the Bulk Egg Generator (previously Bulk Importer) [here](https://github.com/CDNRae/pkhex-bulk-egg-generator) will make it so you can generate all your eggs in PKHeX. I've restored the website so you can try using it (https://pokemon-egg-generator.herokuapp.com/index) along with the <a href="https://github.com/CDNRae/pkhex-bulk-egg-generator/releases/tag/v1.1.0">latest Bulk Importer release</a>. If the website looks like it breaks after you try to generate Pokemon, refresh and try again -- there's some data somewhere that it doesn't like, but it will throw eggs out for you eventually. The Bulk Importer is a bit of a mess but you might be able to get it to work.

**Comment by mrmanyk** [comment]
Wow! Well thanks for the prompt response. Too bad it's not ready but that sounds exciting. I do Soul Link Nuzlockes with my business partner on Twitch.com/slapdashstreams (slapdashstudios.com) and we've wanted to do an egglocke for a while but our podcast audience (D&D Comedy) isn't as into pokemon as we are so getting the necessary donations to make that work is a pipedream. 
Rest assured, I'll be keeping an eye out for this tool. 
I appreciate your efforts immensely and look forward to see what you come out with!

Thanks, 
Zach
